README.md

```diff
@@ -92,7 +92,7 @@ This repository includes a collection of advanced and curated Modules consisting
 | [PolicyAssignment](https://github.com/Azure/ResourceModules/tree/main/arm/Microsoft.Authorization/policyAssignments) |  | [![Authorization: Policyassignments](https://github.com/Azure/ResourceModules/actions/workflows/ms.authorization.policyassignments.yml/badge.svg)](https://github.com/Azure/ResourceModules/actions/workflows/ms.authorization.policyassignments.yml) | [![Deploy to Azure](/docs/media/deploytoazure.svg?sanitize=true)](<https://portal.azure.com/#create/Microsoft.Template/uri/https%3a%2f%2fraw.githubusercontent.com%2fAzure%2fResourceModules%2fmain%2farm%5cMicrosoft.Authorization%5cpolicyAssignments%2fdeploy.json>) |
 | [PrivateDnsZones](https://github.com/Azure/ResourceModules/tree/main/arm/Microsoft.Network/privateDnsZones) |  | [![Network: Privatednszones](https://github.com/Azure/ResourceModules/actions/workflows/ms.network.privatednszones.yml/badge.svg)](https://github.com/Azure/ResourceModules/actions/workflows/ms.network.privatednszones.yml) | [![Deploy to Azure](/docs/media/deploytoazure.svg?sanitize=true)](<https://portal.azure.com/#create/Microsoft.Template/uri/https%3a%2f%2fraw.githubusercontent.com%2fAzure%2fResourceModules%2fmain%2farm%5cMicrosoft.Network%5cprivateDnsZones%2fdeploy.json>) |
-| [PrivateEndpoints](https://github.com/Azure/ResourceModules/tree/main/arm/Microsoft.Network/privateEndpoints) |  | [![Network: Privateendpoints](https://github.com/Azure/ResourceModules/actions/workflows/ms.network.privateendpoints.yml/badge.svg)](https://github.com/Azure/ResourceModules/actions/workflows/ms.network.privateendpoints.yml) | [![Deploy to Azure](/docs/media/deploytoazure.svg?sanitize=true)](<https://portal.azure.com/#create/Microsoft.Template/uri/https%3a%2f%2fraw.githubusercontent.com%2fAzure%2fResourceModules%2fmain%2farm%5cMicrosoft.Network%5cprivateEndpoints%2fdeploy.json>) |
+| [PrivateEndpoints](https://github.com/Azure/ResourceModules/tree/main/arm/Microsoft.Network/privateEndpoints) | :heavy_check_mark: | [![Network: Privateendpoints](https://github.com/Azure/ResourceModules/actions/workflows/ms.network.privateendpoints.yml/badge.svg)](https://github.com/Azure/ResourceModules/actions/workflows/ms.network.privateendpoints.yml) | [![Deploy to Azure](/docs/media/deploytoazure.svg?sanitize=true)](<https://portal.azure.com/#create/Microsoft.Template/uri/https%3a%2f%2fraw.githubusercontent.com%2fAzure%2fResourceModules%2fmain%2farm%5cMicrosoft.Network%5cprivateEndpoints%2fdeploy.json>) |
```

Comment:
I guess we should remove the deploy to azure button if deploy.json is deleted

Reply:
True, and I'll update it. However, I'd not focus too much on the table as we have the auto-generating pipeline ready in the fork

Reply:
done